.github/workflows/phoenix/build.sh

```
@@ -0,0 +1,20 @@
+if [ "$job_device" = "gpu" ]; then
+  build_opts+="--gpu"
+  if [ "$job_interface" = "acc" ]; then
+      build_opts+=" acc"
+  elif [ "$job_interface" = "omp" ]; then
+      build_opts+=" mp"
+  fi
+fi
+
+. ./mfc.sh load -c p -m $job_device
+
+./mfc.sh build --deps-only -j $(nproc) $build_opts
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)